lib/databaseCommand/Config.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace SimpleSAML\Module\perun\databaseCommand;
+
+use SimpleSAML\Configuration;
+
+class Config
```

Comment:
A bit too overused name for the class. Perhaps it should more reflect what the config relates to.